cli/helpers/sca_code_scanner.py

```diff
@@ -10,11 +11,14 @@
 BUILD_GRADLE_DEP_TREE_TIMEOUT = 180
 
 
-def run_pre_scan_actions(documents_to_scan: List[Document], is_git_diff: bool = False):
+def run_pre_scan_actions(context: click.Context, documents_to_scan: List[Document], is_git_diff: bool = False):
+    is_monitor = context.params.get('monitor')
+    project_path = context.params.get('path')
```

Comment:
check if not null

Reply:
in all flows that we are calling it it's not null